src/sentry/runner/commands/presenters/slackpresenter.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+
+import requests
+
+from sentry.runner.commands.presenters.optionspresenter import OptionsPresenter
+from sentry.utils import json
+
+
+class SlackPresenter(OptionsPresenter):
+    def __init__(self, dry_run) -> None:
+        self.drifted_options = []
+        self.channel_updated_options = []
+        self.updated_options = []
+        self.set_options = []
+        self.unset_options = []
+        self.error_options = []
+        self.dry_run = dry_run
+
+    def flush(self):
+        json_data = {
+            # todo: how to grab the region and user responsible.
+            # env variables?
+            "region": None,
+            "user_responsible": None,
```

Comment:
You do not have access to these two in the configoptions,. Please remove them for now. 
We will have to change the flow to enable this.